mod/csv_import/set/abstract/import/01_table_row.rb

```diff
@@ -104,7 +103,7 @@ def respond_to_missing? method, _include_private=false
   end
 
   def field_from_method method
-    method =~ /^(\w+)_field$/ ? $1 : nil
+    method =~ /^(\w+)_field$/ ? Regexp.last_match(1) : nil
```

Comment:
I'm not in love with last_match, but I can't figure out how to do this common pattern much more cleanly.

```
m = method.match /^(\w+)_field$/
m && m[1]
```
I wish this worked: `method.match(/^(\w+)_field$/)&[1]

Reply:
how about `method.match(/^(\w+)_field$/)&.captures.at(0)`?

Reply:
it's actually more readable than my "wish" but probably less so than either the current code or my attempt. 

We could also add our own method to String:
```
def nth_match n, regexp
...

method.nth_match 1, /^(\w+)_field$/
```

Reply:
I don't like that you first say in what part of the match you are interested in and then you define and do the match. 
We could add `def capture n` to MatchData instead:
`method.match(/^(\w+)_field$/)&.capture(0)`

Reply:
https://github.com/decko-commons/decko/pull/129/files